package.json

```
@@ -2,22 +2,36 @@
   "name": "scarf-postgres-exporter",
   "version": "1.0.0",
   "description": "Script to export data from Scarf into any PostgreSQL database",
+  "keywords": [],
+  "author": "Scarf",
+  "license": "Apache 2.0",
   "main": "index.js",
+  "engines": {
+    "node": ">=20.0.0 <21.0.0"
+  },
+  "scripts": {
+    "buildAndRun": "npx tsc && node index.js",
+    "lint": "eslint . --ext .ts",
+    "lint:cache": "eslint . --ext .ts --cache",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
   "dependencies": {
-    "@scarf/scarf": "^1.1.1",
-    "typescript": "^4.9.4"
+    "@scarf/scarf": "1.1.1",
```

```suggestion
    "@scarf/scarf": "^1.3.0",
```

I'm not sure I'm sold on fully pinning dependencies that we have full control over. I'd rather users get updates we push out, as we need to test against breakages before we cut releases anyway. 

Also the latest stable version for @scarf/scarf is `1.3.0`.